HISTORY.md

```diff
@@ -1,3 +1,109 @@
+# TileDB v2.26.0 Release Notes
+
+## Breaking behavior
+
+* The superbuild architecture of the build system has been removed and TileDB is a regular CMake project. Build commands of the form `make && make -C tiledb <targets>` will have to be replaced by `make <targets>`. [#5021](https://github.com/TileDB-Inc/TileDB/pull/5021)
+
+## Breaking C API changes
+
+* All deprecated C and C++ APIs were removed. [#5146](https://github.com/TileDB-Inc/TileDB/pull/5146)
+
+## New features
+
+* Enable array open v2 and query v3 by default. [#5207](https://github.com/TileDB-Inc/TileDB/pull/5207)
+* Add support for getting the datatype of a ndrectangle dimension. [#5229](https://github.com/TileDB-Inc/TileDB/pull/5229)
+* Add dim num support for ndrectangle. [#5230](https://github.com/TileDB-Inc/TileDB/pull/5230)
+
+## Improvements
+
+* Output of `schema.dump()` in TileDB-Py is not captured by Jupyter. Replacing `dump` functions with `operator<<` overloads will give the ability to print the resulted string from Python. [#5026](https://github.com/TileDB-Inc/TileDB/pull/5026)
```

Comment:
```suggestion
* Added new functions to write schema dump to a string rather than stdout, so that non-console interactive environments such as Jupyter will be able to capture and print the output [#5026](https://github.com/TileDB-Inc/TileDB/pull/5026)
```

This should also go in the C API section with the new C API name.